site/src/components/CopyButton/CopyButton.tsx

```diff
@@ -29,51 +28,53 @@ export const CopyButton: React.FC<React.PropsWithChildren<CopyButtonProps>> = ({
   buttonClassName = "",
   tooltipTitle = Language.tooltipTitle,
 }) => {
-  const styles = useStyles();
   const { isCopied, copy: copyToClipboard } = useClipboard(text);
 
+  const fileCopyIconStyles = css`
```

Comment:
Thoughts on these style rules being at the top of the component vs at the bottom?

Reply:
I think I prefer to define things top to bottom, but I'm fine with anything.

Another question, though: is there any difference between having the `css` tag function defined inside the component vs outside?

Reply:
I've always put stateful logic (hooks, contexts, queries) at the top of the file while moving CSS to the bottom, but I don't have a strong opinion, just a habit. I'm happy with either. Would be nice to stay consistent either way we go.

Reply:
yeah, I like top-to-bottom. my instinct is always to scroll up for stuff like this, and I'm always a little sad when I realize I was supposed to scroll down. :p

Reply:
although that's probably less of an issue here than it was with `makeStyles`, because "go to definition" will actually work now 🙃